CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ the requirements below.
     > ...
     ```
 6. Check whether all the generate artifacts in-place properly.
-7. Update [plugin index](https://github.com/drone/drone-plugin-index/blob/master/content/meltwater/drone-cache/index.md) using [DOCS](./DOCS.md).
+7. (OBSOLETE) Update plugin index using [DOCS](./DOCS.md).
```

Comment:
Did you remove this from the CI pipeline?  Believe it would cause a failure on merge.

Reply:
Yep, just making sure that it was not part of the push